.circleci/config.yml

```diff
@@ -137,17 +137,12 @@ jobs:
           path: coverage
 
   build_docs:
+    resource_class: medium+
     <<: *container_config
     steps:
-      - *restore_repo
-      - *sum_lockfiles
-      - *restore_npm_cache
-      - *npm_install
-      - *generate_base_path
+      - *attach_workspace
       - run:
           command: node -v && node -e 'console.log(`heap_size_limit = ${v8.getHeapStatistics().heap_size_limit / (1024 * 1024)} MB`)' && npm run build -- --base "/${BASE_PATH}/"
-          environment:
-            NODE_OPTIONS: --max_old_space_size=1400
```

Comment:
can you add a little bit more detail to the PR description about this change? You mentioned heap size but we should be explicit about this one.

Reply:
Added another bullet point!